### ISSUE TITLE
Is DIANN using the wrong setting for Cys carbamidomethylation

### ISSUE THREAD

**Issue by jossmi** [vdemichev/DiaNN#618]
I am doing some analysis on samples that have not been reduced and alkylated (unusual, I know), and so I have unchecked the "Cys carbamidomethylation" option. However, when I run DIANN, I saw that the log says "Cysteine carbamidomethylation enabled as a fixed modification". See attached for a screenshot. 

Is the log correct and DIANN is ignoring that I turned that modification off, or is that some bug in the log?

Thanks,
Josh

![DIANN_CysAlkylationError](https://user-images.githubusercontent.com/77409436/218145870-acaef109-1c33-470a-9e70-c0fe0310d014.png)

**Comment by vdemichev** [comment]
Hi Josh,

Strange, I cannot reproduce this on my PC. Can you please try again, by creating two pipeline steps, one with C(cam) and one without, and then saving the log of running the entire pipeline? In this case you don't need any raw files and can use a small compact FASTA.

Unrelated comment:
- FASTA should not contain any decoys for regular analyses (unless you specifically want this for manual validation of FDR using R/Python). In any case, DIA-NN will not recognise them but will instead treat them the same as regular proteins.
- I would not use double pass.
- I would fix mass accs & scan window.
- Missed cleavage setting & variable mods will be suboptimal for most analyses.
- Library generation set to "IDs, RT and IM profiling" will likely be faster and maybe very slightly better.

Best,
Vadim

**Comment by jossmi** [comment]
Thanks for your feedback. I'll perform that test and let you know.

Regarding the mass accuracies and scan window settings, I left them at "0" because I was under the impression that allowed DIANN to optimize those settings. It seemed that would be preferable to me overriding them with some value I determined, or to some threshold. Could you provide a bit more detail on proper adjustment of those settings?

**Comment by vdemichev** [comment]
Yes, with mass acc usually can do this very quickly using a small spectral library.

**Comment by jossmi** [comment]
Ok, so why would I not leave them at 0? What do you mean by "fix"? And scan window?

**Comment by vdemichev** [comment]
Fix = use a specific value. Scan window the same - see what DIA-NN recomments for several runs & put in the average

**Comment by vdemichev** [comment]
Please see the docs on general guidelines on settings, when to fix mass accuracies, etc
